index.html

```diff
@@ -66,6 +80,175 @@ <h1 class="header__title">I'm Gladwin Glad to see<br />you!</h1>
+            </ul>
+            <p class="desktop__project__description">
+              Exploring the future of media in Facebook's first Virtual Reality
+              app; a place to discover and enjoy 360 photos and videos on Gear
+              VR.
+            </p>
+            <ul class="desktop__tags">
+              <li class="desktop__tag__language">
+                <a href="#" class="desktop__language">html</a>
+              </li>
+              <li class="desktop__tag__language">
+                <a href="#" class="desktop__language">Ruby on rails</a>
+              </li>
+              <li class="desktop__tag__language">
+                <a href="#" class="desktop__language">css</a>
+              </li>
+              <li class="desktop__tag__language">
+                <a href="#" class="desktop__language">javaScript</a>
+              </li>
+            </ul>
+            <button type="button" class="btn4">
+              <a href="#" class="see__project">See Project</a>
+            </button>
+          </div>
+          <img class="hand__cursor" src="./images/hand-cursor 1.png" alt="" />
+        </div>
+      </div>
+
+      <div class="desktop__works2">
+        <div class="desktop__cards">
+          <img class="img2" src="./images/Snapshoot Portfolio4.svg" alt="" />
+          <div class="desktop__clients__info2">
+            <h class="desktop__project__title">Uber Navigation</h>
+            <ul class="desktop__client">
+              <li class="desktop__project__name">Uber</li>
+              <li class="desktop__counter">
+                <img src="./images/Counter.svg" alt="" />
+              </li>
+              <li class="desktop__project__role">Lead Developer</li>
+              <li class="desktop__counter">
+                <img src="./images/Counter.svg" alt="" />
+              </li>
+              <li class="desktop__year">2015</li>
+            </ul>
+            <p class="desktop__project__description">
+              A smart assistant to make driving more safe, efficient, and fun by
+              unlocking your most expensive computer: your car.
+            </p>
+            <ul class="desktop__tags">
+              <li class="desktop__tag__language">
+                <a href="#" class="desktop__language">html</a>
+              </li>
+              <li class="desktop__tag__language">
+                <a href="#" class="desktop__language">Ruby on rails</a>
+              </li>
+              <li class="desktop__tag__language">
+                <a href="#" class="desktop__language">css</a>
+              </li>
+              <li class="desktop__tag__language">
+                <a href="#" class="desktop__language">javaScript</a>
+              </li>
+            </ul>
+            <button type="button" class="btn4">
+              <a href="#" class="see__project">See Project</a>
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+
```

Comment:
The margin below the works section that separates the works section from the about me section is bigger than the original design as shown below :point_down: Kindly update it accordingly in order to match the actual design. 

### Your design 

![Screenshot from 2023-02-24 08-03-58](https://user-images.githubusercontent.com/65233860/221126313-b071e22b-d662-47d0-b1dc-a59f456d2670.png)


### Figma design


![Screenshot from 2023-02-24 08-04-13](https://user-images.githubusercontent.com/65233860/221126326-a1206f3b-3b9f-40a5-9ff1-918ee9f1788e.png)